index.html

```diff
@@ -638,6 +638,407 @@ <h5 class="mw-100 mt-3">START YOUR DAY WITH OUR TOP 5 ARTICLES</h5>
                 </div>
                 </section>
                 </main>
+                 <!------------Milestone 3--------->
+            <div class="container mt-5">
+                <h5 class="mw-100 text-center border-bottom text-dark text-decoration-underline font-weight-bolder"><img class="p-2" src="img/favicon-32x32.png" alt="" />IN THE MAGAZINE</h5>
+                <div class="row">
```

Comment:
### Spacing
- [ ] Let's be mindful of the spacings on the rest of the contest. They all should be having a uniform margin as the page need not be covering the whole view width of the screen bootstrap `contani3er` class can help us achieve this, do you mind implementing thin?

|Source Page|Clone Page|
|:--:|:--:|
|![source](https://user-images.githubusercontent.com/22345961/114778489-961eb600-9d7d-11eb-8e45-30386c41c3d0.png)|![clone](https://user-images.githubusercontent.com/22345961/114778507-9dde5a80-9d7d-11eb-8aec-235333d219cd.png)|